sncosmo/builtins.py

```diff
@@ -710,6 +710,25 @@ def load_mlcs2k2(relpath, name=None, version=None):
                          args=('models/mlcs2k2/mlcs2k2.modelflux.v1.0.fits',),
                          version='1.0', meta=meta)
 
+
+# SNEMO
+def load_snemomodel(relpath, name=None, version=None):
+    abspath = DATADIR.abspath(relpath)
+    return SNEMOSource(abspath, name=name, version=version)
+
+for name, file, ver in [('snemo2', 'snemo2_ev.dat', '1.0'),
+                        ('snemo7', 'snemo7_ev.dat', '1.0'),
+                        ('snemo15', 'snemo15_ev.dat', '1.0')]:
```

Comment:
It seems like it makes sense to call these `'snemo'` version `'2'` (or `'2.0'`), `'7'` and `'15'` rather than version `'1.0'` of three different models.

EDIT: Ah I see now that they're sort of different models.